modules/tinymce/src/plugins/searchreplace/test/ts/module/test/Utils.ts

```diff
@@ -62,6 +62,10 @@ const pSelectPreference = async (editor: Editor, name: string): Promise<void> =>
   TinyUiActions.clickOnUi(editor, '.tox-selected-menu[role=menu] div[aria-label="' + name + '"]');
 };
 
+const getFindInputSelector = Fun.constant('.tox-bar > .tox-form__group > input.tox-textfield');
+
+const getReplaceWithInputSelector = Fun.constant('.tox-form > .tox-form__group > input.tox-textfield');
```

Comment:
This is too brittle. It's not something we need to fix now, but I don't want to forget about it.

Please log a JIRA ticket - labeled `tech debt` - to add some sort of unique class or attribute to elements based on their component name so tests can find them. We made this change to toolbar buttons in TinyMCE 7, I think it needs to become more widespread.

Reply:
Ticket created: TINY-11008 ✅